src/devoApp/__tests__/DevoAppBase.tests.ts

```diff
@@ -75,7 +76,20 @@ const userInfoResponse = new Promise<UserInfo>((resolve) => {
 describe('DevoApp Base Test', () => {
   it('It returns query client', async () => {
     const devoApp = new DevoAppTest(userInfoResponse);
-    const queryClient = await devoApp.getQueryClient();
+    const appInfo: AppInfo = {
+      application: 'app1',
+      component: 'cmp1',
+    };
+    const queryClient = await devoApp.getQueryClient(appInfo);
+    expect(queryClient).toBeInstanceOf(QueryClientMock);
+  });
+
+  it('It returns query client with app info', async () => {
```

Comment:
as `appInfo` is required now, these both tests check the same thing, therefore are duplicated 👀 

we could delete `It returns query client with app info` test case

Reply:
Your're right. Solved.